benchmarks/squad/squad.py

```diff
@@ -0,0 +1,249 @@
+import re
+import string
+
+from typing import List, Tuple, Union
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment  # type: ignore
+
+from inspect_ai import Task, task
+from inspect_ai.dataset import Sample, hf_dataset
+from inspect_ai.scorer import Score, Target, bootstrap_std, mean, scorer
+from inspect_ai.solver import system_message, generate, TaskState
+
+
+SYSTEM_MESSAGE = """
+Using only the context provided in the question itself, you are tasked with answering a question in as few words as possible.
+"""
+
+@task
+def squad():
+   
+    dataset = hf_dataset(
+        path="rajpurkar/squad",
+        split="validation",
+        sample_fields=record_to_sample,
+        trust=True,
+        shuffle=True
+    )
+
+    return Task(
+        dataset=dataset,
+        plan=build_plan(),
+        scorer=squad_f1_scorer()
+    )
+
+def record_to_sample(record):
+  return Sample(
+      input=format_input(record),
+      target=format_target(record),
+      id=record["id"],
+  )
+
+def format_input(record) -> str:
+  passage_str = f"""Context: {record["context"]}"""
+  question_str = f"""Question: {record["question"]}"""
+  input_str = "\n".join([passage_str, question_str])
+
+  return input_str
+
+
+def format_target(record) -> List[str]:
+  answers = record["answers"]
+  choices = answers["text"]
+
+  target = choices
+  return target
+
+def build_plan() -> List:
```

Comment:
I think this `build_plan()` function is simple enough that it should just be inlined in the task definition.

Reply:
Good point! Resolved